bugwarrior/services/phab.py

```diff
@@ -97,13 +133,13 @@ def issues(self):
 
         log.info("Found %i issues" % len(issues))
 
+        projects = fetch_issue_metadata(self.api, issues)
+
         for phid, issue in issues:
 
-            project = self.target  # a sensible default
-            try:
-                project = projects.get(issue['projectPHIDs'][0], project)
-            except IndexError:
-                pass
+            # TODO fix this now that we have access to proper projects.
+            # It was bugged before
+            project = self.target
```

Comment:
The previous author had left in some code that was unused (because the projects list was always empty), anticipating that he could use it after populating the projects list. It should now be possible to implement the feature he planned to implement, though not using his code. So I just deleted his code (which would now break things instead of do nothing) and added a TODO marker.